apps/frontend/src/app/Solver/GOSolver/solveLP_simplex.ts

```diff
@@ -0,0 +1,119 @@
+/**
+ * Solve a Linear Program defined by:
```

Comment:
There are some places where `i` and `j` seems to follow some convension (`i` for row, `j` for column?). If so, add a note for that at the beginning of the file. Maybe switch to `r` and `c` to make it more "intuitive"?